File: C/019.c
// C program: Divisible by 5 and 8

#include<stdio.h>
int main()
{
	int b;
	printf("Enter an integer = ");
	scanf("%d",&b);

	if(b%5==0 && b%8==0)
	{
		printf("%d is divisible by both 5 and 8",b);
	}
	else
	{
		printf("%d is not divisible by both 5 and 8",b);
	}
}

File: Java/019.java
// Java class: Divisible by four or eight

package projectFolder;

import java.util.Scanner;

public class DivisibleByFourOrEight
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter any integer = ");
        int num = scan.nextInt();
        if (num % 4 == 0 || num % 8 == 0)
        {
            System.out.println("The integer is divisible by either 4 or 8");
        }
        else
        {
            System.out.println("The integer is divisible by neither 4 nor 8");
        }
    }
}